app/server/models/organisation.py

```diff
@@ -58,12 +58,12 @@ class Organisation(ModelBase):
     def master_organisation() -> "Organisation":
         return Organisation.query.filter_by(is_master=True).first()
 
-    def send_welcome_sms(self, to_user):
+    def send_welcome_sms(self, to_user: dict):
         if self.custom_welcome_message_key:
             message = i18n_for(to_user, "organisation.{}".format(self.custom_welcome_message_key))
         else:
             message = i18n_for(to_user, "organisation.generic_welcome_message")
-        message_processor.send_message(to_user.phone, message)
+        message_processor.send_message(to_user.get('phone'), message)
```

Comment:
`to_user` is a dict, so needs to use `.get(`

Reply:
wait are you sure? tests are now failing:
```
AttributeError: 'User' object has no attribute 'get'
```

Reply:
oh it's because my test makes a bad assumption